extensions/sitemap/__init__.py

```diff
@@ -0,0 +1,222 @@
+#
+# sitemap - A fork of the sphinx_sitemap extension with parallel read and write support enabled.
+# Based on a sphinx_sitemap PR: https://github.com/jdillard/sphinx-sitemap/pull/29
+#
+# Copyright (c) 2013 Michael Dowling <mtdowling@gmail.com>
+# Copyright (c) 2017 Jared Dillard <jared.dillard@gmail.com>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+import queue
+import os
+import xml.etree.ElementTree as ET
+
+from multiprocessing import Manager
+
+try:
+    from sphinx.util.logging import getLogger
+
+    logger = getLogger(__name__)
+
+
+    def error(_, message):
+        logger.error(message)
+
+
+    def warn(_, message):
+        logger.warning(message)
+
+
+    def info(_, message):
+        logger.info(message)
+except ImportError:
+    def error(app, message):
+        app.warn(message, prefix='ERROR: ')
+
+
+    def warn(app, message):
+        app.warn(message)
+
+
+    def info(app, message):
+        app.info(message)
+
+
+def setup(app):
+    """Setup connects events to the sitemap builder"""
+    app.add_config_value(
+        'site_url',
+        default=None,
+        rebuild=''
+    )
+    app.add_config_value(
+        'sitemap_url_scheme',
+        default="{lang}{version}{link}",
```

Comment:
@justinegeffen - This is where we can customize the content elements included in the generated ``sitemap.xml`` file. Removing ``{verison}`` here removes the ``version`` value set via the ``conf.py`` file.